features/git-town-sync/all_branches/conflicts/feature_branch_merge_main_branch_conflict/without_tracking_branch_updates/last_branch.feature

```diff
@@ -57,7 +57,7 @@ Feature: git-town sync --all: handling merge conflicts between feature branch an
 
 
   Scenario: skipping
-    When I run `git-town sync --skip`
+    When I run `git-town continue --skip`
```

Comment:
How do you feel about `git-town skip` here?

Reply:
I changed it when trying to describe what skip does. I had to put the word `continue` in the description so I thought it made sense to join them. Git does have the precedence of `--skip` in rebase so I agree with changing it back. I like there description in git rebase:
* `--continue`: Restart the rebasing process after having resolved a merge conflict.
* `--skip`: Restart the rebasing process by skipping the current patch.